Add Quickview to Suite

tbarsballe opened the pull request.

nstires-boundless commented:
Looking at this PR, I think we're pulling from too far ahead. I think this is what you mentioned in Slack, but I misread! The newer version of sdk-apps doesn't work.

We've been working off the branch "suite-master" which has the latest commit of 32a5ede089bdc839b4e5eeea7449288371ead434

It's mostly pinned to an older version of Quickview, with largely packaging updates. There's also a a mysterious commit I made here that would change the actual artifact. https://github.com/boundlessgeo/sdk-apps/commit/1ee51a8d8f12738d8240e614e8fc1d1908224f51

All of the rpmbuild/ dir on the top level of the repo is slated for removal imminently (post-suite repo migration today hopefully).

tbarsballe commented:
Moved to suite-master branch, and updated ant build accordingly.

After the suite packaging stuff gets safely moved over to the suite repo, we should probably look into  updating quickview to master and seeing if we can get it working with the latest sdk-apps.
